serverlessworkflow/modules/ROOT/pages/cloud/kubernetes-service-discovery.adoc

```diff
@@ -0,0 +1,212 @@
+= Kubernetes Service Discovery for Serverless Workflow Application Configuration
+:compat-mode!:
+// Metadata:
+:description: Explain what is and how the service discovery works
+:keywords: kogito, workflow, quarkus, serverless, service-discovery, enhanced-service-discovery
+// links
+:quarkus_issue_url: https://github.com/quarkusio/quarkus/issues/27457
+
+
+The Kubernetes Service Discovery allows you to have a static URI that defines a Kubernetes resource that will be used
+to perform HTTP requests. The resource defined in the URI will then be queried in the current Kubernetes cluster and
+translated to a valid URL. The service discovery feature works during the Application startup, it scans all Quarkus
```

Comment:
```suggestion
translated to a valid URL. The service discovery feature works during the Application startup: it scans all Quarkus
```

Reply:
I just changed the comma to `:` as it makes more sense to the structure of the sentence IMO

Reply:
I am not sure about it, can't see why to use ":" in there.

Reply:
I think we can wait for @hmanwani-rh's review.